scripts/cent6_jswarm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+CINCH="$(readlink -f "$(dirname "$0")/../")"
+CINCH_VERSION=$(grep "${CINCH}/setup.py" -e 'version=' | sed -e "s/.*version='\(.*\)'.*/\1/")
+
+echo "*****************************************************"
+echo "Building cinch container for version ${CINCH_VERSION}"
+echo "*****************************************************"
+
+echo "Starting container"
+ansible -i /dev/null localhost -m docker_container -a "image=greghellings/centos_base:6 name=jswarm detach=true tty=true command=/bin/bash"
```

Comment:
In this case, I'm not storing the resulting image in my personal account, rather I'm using one of my personal images as a base. That image contains CentOS with yum update and yum install sudo already executed. Doing this allows me to avoid installing sudo from Ansible, where it's just spurious noise that is already present in all machine based CentOS installs.

Of course, I could also move the installation of such software into this script using ansible ad hoc commands. I had done things this way because my home Internet connection is incredibly slow, so I wanted to have as much of the software pre-installed as possible. On the hosts (e.g. Travis, Jenkins, etc) that we're going to be using, such bandwidth limitations are almost non-existent, so I'm not at all opposed to moving this to a base CentOS image pull.